website/pages/docs/plugins/federation.mdx

```diff
@@ -265,7 +264,7 @@ builder.asEntity(Media, {
 
 See federation documentation for more details on `interfaceObject`s
 
-### composeDirective
+### composeDirective =
```

Comment:
Is this `=` a typo?

Reply:
Yup, not sure where that come from

Reply:
Looks like it was already in the Readme, which I copied here to keep it in sync